### DISCUSSION TITLE
Add possibility to use custom Image component

### DISCUSSION THREAD

**Discussion by sandergo90** [GetStream/stream-chat-react-native#1030] [Ideas]
Hi again :) 

We are using end-to-end encryption in our chat implementation.
For encrypting and decrypting text messages we can easily do this using the `MessageText` component and the `doSendMessageRequest` props. 

When uploading an image, the encryption part can also be done easily. But for decryption, we would have to override the full Gallery component which we rather not do. It would be nice to be able to provide a custom Image component to the Gallery. 

Can this be added to the backlog please?

Kind regards,
Sander

**Comment by madsroskar** [comment]
Hi, @sandergo90! We have a task on our backlog to improve customization related to the Gallery, and will provide a way to override the `GalleryImage` component used within the Gallery as part of that.

I don't know yet exactly when we will be able to start work on it, but I'll make a note to update this thread when we know more about that.

For internal tracking: CRNS-484
